README.md

```diff
@@ -126,6 +126,20 @@ cd build
 cmake .. -G "MinGW Makefiles"
 cmake --build . --target install
 ```
+## Building WASM (linux only)
+Compiling `mgclient` requires the Emscripten sdk found in https://github.com/emscripten-core/emsdk and OpenSSL WASM.
+  - Clone the emsdk repo above and do `./emsdk install latest && ./emsdk activate latest && source ./emsdk_env.sh`
+  - Clone openssl repo found in https://github.com/openssl/openssl and compile it to WASM with `emconfigure ./Configure -no-asm -no-tests && make -j8`
```

Comment:
This doesn't work as-is, some weird path is prepended, the standalone script having all this is definitely required 😄

Reply:
now it's fully automated ;)